apollo/agent/models.py

```diff
@@ -88,5 +94,15 @@ class AgentHealthInformation:
         metadata=config(exclude=exclude_none_values), default=None
     )
 
-    def to_dict(self) -> Dict:  # type: ignore
-        pass
+
+@dataclass
+class AgentExecuteSqlQueryResponse(DataClassJsonMixin):
```

Comment:
Opted to declare this here as a 'built-in command response schema', so it could be easier to adhere to in the different clients.